site_scons/site_tools/doxygen.conf

```diff
@@ -0,0 +1,2658 @@
+# Doxyfile 1.9.1
```

Comment:
why do we need this in site_tools? if this is a template, it should live in the templates dir